gensim/models/flsamodel.py

```diff
@@ -1473,18 +1358,15 @@ def get_word_embedding(
                     Maximum distance between the current and predicted word within a sentence.
                 vector_size : int
                     Dimensionality of the word vectors.
-                workers : int
-                    Use these many worker threads to train the model
-                    ( = faster training with multicore machines).
         """
         self.model = Word2Vec(
             sentences=data,
             vector_size=vector_size,
             window=window,
             min_count=min_count,
-            workers=workers,
         )
         # FIXME is the whole Word2Vec model really needed? Why are we storing it as an attribute?
+        #ERijck: This way users can do a post analysis. However, it is not crucial and can be removed.
```

Comment:
OK. Best add a comment to that effect: 

`# Storing the whole trained model is not needed. We're only doing it so users can do a post analysis. Could be removed.`